invenio_rdm_records/resources/resources.py

```diff
@@ -222,6 +222,38 @@ def remove(self):
         return response, 200
 
 
+class RDMRecordRequestsResource(ErrorHandlersMixin, Resource):
+    """Record communities resource."""
+
+    def __init__(self, config, service):
+        """Constructor."""
+        super().__init__(config)
+        self.service = service
+
+    def create_url_rules(self):
+        """Create the URL rules for the record resource."""
+        routes = self.config.routes
+        url_rules = [
+            route("GET", routes["list"], self.search),
+        ]
+        return url_rules
+
+    @request_search_args
+    @request_view_args
+    @response_handler(many=True)
+    def search(self):
+        """Search for record's communities."""
+        items = self.service.search(
+            identity=g.identity,
+            params=resource_requestctx.args,
+            record_pid=resource_requestctx.view_args["record_pid"],
+            search_preference=search_preference(),
+            expand=resource_requestctx.args.get("expand", True),
```

Comment:
Moderate: I think that you are missing to add the expand to to the search decorators. In records resources we do add it with the [request_extra_args](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/resources/records/config.py#L40).